backend/api/views.py

```diff
@@ -99,8 +102,28 @@ def metric_by_id(request, exp_id, metric_id):
 
 
 @api_view(['POST'])
-def uptake(request):
+def enrollment(request):
     """
-    Take a JSON payload from telemetry streaming to record experiment uptake.
+    Take a JSON payload from telemetry streaming to record experiment enrollment.
     """
+    payload = request.data.get('enrollment')
+    if not payload or not isinstance(payload, list):
+        raise ValidationError
+
+    for data in payload:
+        window_start = timezone.make_aware(
+            datetime.datetime.fromtimestamp(data['window_start'] / 1000))
+        window_end = timezone.make_aware(
+            datetime.datetime.fromtimestamp(data['window_end'] / 1000))
+
+        Enrollment.objects.create(
```

Comment:
Just occurred to me we could create an array of formatted dictionaries and then use 

https://docs.djangoproject.com/en/2.0/ref/models/querysets/#bulk-create

to reduce the number of round trips to the db, but that's probably an optimization we can add later